### DISCUSSION TITLE
Releases available as Dockerfiles?

### DISCUSSION THREAD

**Discussion by fBedecarrats** [posit-dev/positron#4521] [Q&A] [answered]
I would love to test the positron IDE on a remote environment (https://www.sspcloud.fr/). Is it available as a dockerfile?

**Comment by jmcphers** [comment, marked as answer]
No, we don't plan to put Positron in a Dockerfile. However, if you spin up the remote environment and make it available via SSH, you can use Positron's Remote SSH feature to run R and Python in the remote environment.

**Comment by fBedecarrats** [comment]
Thank you for your reply!
